Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -311,6 +311,13 @@ protected theorem pow_one : M ^ 1 = M := by
 protected theorem pow_succ' {n : ℕ} (h : n ≠ 0) : M ^ (n + 1) = M * M ^ n := by
   rw [add_comm, M.pow_add h, Submodule.pow_one]
 
+@[simp]
+theorem bot_pow {n : ℕ} (hn : n ≠ 0) : (⊥ : Submodule R A) ^ n = ⊥ := by
```

Comment:
Just a remark that [zero_pow](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupWithZero/Basic.html#zero_pow) isn't applicable here since we only have a [Submodule.instNonUnitalSemiring](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Algebra/Operations.html#Submodule.instNonUnitalSemiring) instance (1 is left identity and not necessarily right identity).